x-pack/legacy/plugins/apm/public/utils/formatters/__test__/duration.test.ts

```diff
@@ -106,11 +122,13 @@ describe('duration formatters', () => {
     it('microseconds', () => {
       const unit = 'microseconds';
       expect(convertTo({ unit, microseconds: 20 })).toEqual({
+        convertedValue: 20,
         unit: 'μs',
         value: '20',
         formatted: '20 μs'
       });
       expect(convertTo({ unit, microseconds: 35 })).toEqual({
+        convertedValue: 35,
         unit: 'μs',
         value: '35',
```

Comment:
I forget: what's the difference between `value` and `convertedValue`?

Reply:
value will returned the number formated:
```javascript
expect(convertTo({ unit, microseconds: oneHourAsMicro })).toEqual({
        convertedValue: 1,
        unit: 'h',
        value: '1.0',
        formatted: '1.0 h'
      });
```